install_pkgs.R

```diff
@@ -0,0 +1,25 @@
+# Create user library and install devtools package
+dir.create(Sys.getenv('R_LIBS_USER'), recursive = TRUE);
+.libPaths(Sys.getenv('R_LIBS_USER'));
+
+# Install devtools and its dependencies
+install.packages('usethis');
+install.packages('covr');
+install.packages('httr');
+install.packages('rversions');
+install.packages('devtools');
+
+# Install packages from requirements list
+requirements_file <- file("r_requirements.txt",open="r")
```

Comment:
Small detail but we may want to allow the user to specify the file rather than hardcode it.